structs/ctl/types.go

```diff
@@ -94,6 +94,44 @@ type CommunityListResponse struct {
 	Communities []string `json:"communities"`
 }
 
+type CommunityPSRequest struct {
+	CommunityID string `json:"community_id"`
+}
+
+type CommunityPSProcess struct {
+	*community.Process
+	Instances []*community.ProcessInstance `json:"instances"`
+}
+
+type CommunityPSResponse struct {
+	Processes []*CommunityPSProcess `json:"processes"`
+}
+
+type CommunityStartProcessRequest struct {
+	CommunityID string `json:"community_id"`
+
+	App   string   `json:"app"`
+	Args  []string `json:"args"`
+	Env   []string `json:"env"`
+	Flags []string `json:"flags"`
+
+	InstancesNodes []string `json:"instance_nodes"`
+}
+
+type CommunityStartProcessResponse struct {
```

Comment:
should we return an optional error here? same for StopProcessResponse?

Reply:
Errors are returned in the `ResponseWrapper` struct, so we get it for free with all response message types